### PR TITLE
ksqldb-latest: fix ksqlDB finding distinct tutorial (part 2)

### PR DESCRIPTION
**Pull request by vcrfxia**
### Description

Another follow-up fix to https://github.com/confluentinc/kafka-tutorials/pull/781. Turns out we need to set broker retention since the derived stream (DISTINCT_CLICKS) doesn't inherit the longer retention policy from the topic created by the windowed aggregation. See https://github.com/confluentinc/kafka-tutorials/issues/695 for context.

### CODE REVIEW

File: _includes/tutorials/finding-distinct/ksql/code/docker-compose.yml [hunk 1]
```diff
@@ -28,6 +28,7 @@ services:
       KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
       KAFKA_TRANSACTION_STATE_LOG_MIN_ISR: 1
       KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR: 1
+      KAFKA_LOG_RETENTION_MS: -1
```
Comment (mjsax) [inline _includes/tutorials/finding-distinct/ksql/code/docker-compose.yml:31]: Just for my own education: is it custom to set infinite retention in tutorials?

**Comment by vcrfxia** [comment]
Minor change that's already been applied on the [0.16 branch](https://github.com/confluentinc/kafka-tutorials/commit/1c523829b769119d2273b6c403049af189ca8f7e). Merging without review since the build is passing.

**Review by mjsax — APPROVED** [review]
LGTM.

File: _includes/tutorials/finding-distinct/ksql/code/docker-compose.yml (hunk 1 above)
Comment (vcrfxia) [reply _includes/tutorials/finding-distinct/ksql/code/docker-compose.yml:31]: No, there is more discussion about how to properly update the tutorials to address the underlying change in behavior in https://github.com/confluentinc/kafka-tutorials/issues/695. The branch targeted by this PR (ksqldb-latest) is used for ksql release validation only, and not live on the tutorials site, which is why we are opting for quicker fixes in the meantime.

**Pull request merged**